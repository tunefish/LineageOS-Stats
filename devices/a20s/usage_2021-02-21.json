{
  "17.1": 2476
}