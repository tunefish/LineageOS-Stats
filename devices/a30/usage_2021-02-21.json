{
  "17.1": 29,
  "18.0": 653,
  "18.1": 217,
  "none": 4
}