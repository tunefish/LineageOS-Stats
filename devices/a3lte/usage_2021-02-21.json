{
  "15.1": 602,
  "16.0": 215,
  "17.1": 678
}