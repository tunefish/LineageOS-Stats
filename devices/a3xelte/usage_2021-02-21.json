{
  "15.1": 113,
  "16.0": 469,
  "17.1": 3633,
  "18.0": 3,
  "18.1": 14,
  "52.8": 1,
  "none": 2
}