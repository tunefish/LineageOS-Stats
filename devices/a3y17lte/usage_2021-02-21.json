{
  "14.1": 5508,
  "15.1": 346,
  "16.0": 1033,
  "17.1": 988,
  "18.0": 1
}