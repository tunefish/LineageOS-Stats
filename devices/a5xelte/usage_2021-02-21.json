{
  "15.1": 57,
  "16.0": 537,
  "17.1": 4349,
  "18.0": 12,
  "18.1": 3,
  "none": 1
}