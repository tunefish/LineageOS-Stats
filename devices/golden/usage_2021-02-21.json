{
  "13.0": 1060,
  "14.1": 3462,
  "15.1": 4,
  "17.1": 5,
  "none": 1
}